dbmongo/lib/urssaf/testData/debitCorrompuTestData.csv

```diff
@@ -2,4 +2,5 @@ Urssaf_gestion;Dep;num_cpte;Siren;Dt_immat;Etat_cpte;Cd_pro_col;Periode;Num_Ecn;
 939;27;111982477292496174;000000000;20150708;5;0;201712;659;9;990505;195907;864497;3;74;0
 238;72;636043216536562844;111111
 838;36;450359886246036238;222222222;20050501;1;1;199904;738;4;1121018;1;2;43;40;1
-
+\",
```

Comment:
Note: pour provoquer `parse error on line 5, column 1: bare \" in non-quoted-field`